roles/ceph-facts/tasks/facts.yml

```diff
@@ -290,6 +290,18 @@
       ntp_service_name: ntpd
     when: ansible_os_family in ['RedHat', 'Suse']
 
+- name: set chrony daemon name RedHat and Ubuntu based OSs
+  block:
+    - name: set chronyd daemon name for RedHat based OSs
+      set_fact:
+          chrony_daemon_name: chronyd
```

Comment:
just a minor thing but the fact name isn't aligned correctly

```yaml
      set_fact:
        chrony_daemon_name: chronyd
```

Reply:
@dsavineau `chrony_service_name` ?

Reply:
no that's about :

```yaml
      set_fact:
-          chrony_daemon_name: chronyd
+        chrony_daemon_name: chronyd
```